docs/visual-basic/language-reference/statements/if-then-else-statement.md

```diff
@@ -73,6 +81,8 @@ If condition Then [ statements ] [ Else [ elsestatements ] ]
   
  `End If`  
  Terminates the `If`...`Then`...`Else` block.  
+
+
   
 ## Remarks  
```

Comment:
lines 89 and 99 should be level-3 heads, not level-2
line 90: space between True and elseifcondition
line 97: several possible values --> more than three possible values
line 100 doesn't actually say what single-line syntax is: a single condition with code to execute if it's true